docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+services:
+  mongodb:
+    image: mongo:latest
+    container_name: mongodbserver
+    hostname: mongodb
+    volumes:
+      - ./src/database/mongod.conf:/etc/mongod.conf
+      - ./src/database/initdb.d/:/docker-entrypoint-initdb.d/
+      - ./src/database/data/db/:/data/db/
+      - ./src/database/data/log/:/var/log/mongodb/
+    ports:
+      - 27017:27017
```

Comment:
can change to 27016:27017
`HOST:CONTAINER`
https://docs.docker.com/compose/compose-file/compose-file-v3/#ports

Reply:
Ports changed to 27016:27017